### PR TITLE
Convert infrastructure file list to an array (#infra)

### PR DESCRIPTION
**Pull request by VladimirSlavik**
Port of #3718

(cherry picked from commit abdad35b5ed76fa6a640a1e02fddca9545dae875)

CC @jstodola

### CODE REVIEW

File: .github/workflows/infra-check.yml [hunk 1]
```diff
@@ -43,7 +43,7 @@ jobs:
           for f in $changed_files; do
             matched="no"
 
-            for infra_f in $INFRASTRUCTURE_FILES; do
+            for infra_f in "${INFRASTRUCTURE_FILES[@]}"; do
```
Comment (jkonecny12) [inline .github/workflows/infra-check.yml:46]: Ahh I hate this hard to understand stuff on bash. Thanks!

**Review by jkonecny12 — APPROVED** [review]
Great improvement, thanks!

**Review by M4rtinK — APPROVED** [review]
Looks good to me as well. :)

File: .github/workflows/infra-check.yml (hunk 1 above)
Comment (VladimirSlavik) [reply .github/workflows/infra-check.yml:46]: Oh, so do I - got it right after three tries.

**Review by jstodola — APPROVED** [review]

**Review by rvykydal — APPROVED** [review]
Thank you

**Pull request merged**